tests/test_countgraph.py

```diff
@@ -1491,3 +1491,25 @@ def test_counting_load_bigcount():
         print(i, count_table.count('ATATATATAT'))
     count = count_table.get('ATATATATAT')
     assert count == 500
+
+
+def test_reverse_bighash():
+    hi = khmer._Countgraph(31, PRIMES_1m)
+    kmer = 'C' * 31
+    hashval = hi.hash('C' * 31)
+    assert hi.reverse_hash(hashval) == kmer
+
+    hi = khmer._Countgraph(33, PRIMES_1m)
+    kmer = 'C' * 33
+    hashval = hi.hash('C' * 33)
+    assert hi.reverse_hash(hashval) != kmer
```

Comment:
For k > 32, this is not so much enforcement as it is verifying our expectations.